### PR TITLE
feat: added issue and PR template, semantic test

### PR DESCRIPTION
**Pull request by iamayushdas**
added

1. PR template
2. issue template
3. semantic test

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+labels: ''
+assignees: ''
+
+---
+
+### Issue description
+
+### Environment
+
+* your apisix-java-plugin-runner version;
```
Comment (spacewander) [inline .github/ISSUE_TEMPLATE/bug-report.md:14]: Please remove the unnecessary ';'

File: .github/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,29 @@
+Please answer these questions before submitting a pull request
+
+- Why submit this pull request?
+- [ ] Bugfix
+- [ ] New feature provided
+- [ ] Improve performance
+- [ ] Backport patches
```
Comment (spacewander) [inline .github/PULL_REQUEST_TEMPLATE:7]: We don't have a plan to do backport yet.

Comment (iamayushdas) [reply .github/PULL_REQUEST_TEMPLATE:7]: okay, fixing it

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (iamayushdas) [reply .github/ISSUE_TEMPLATE/bug-report.md:14]: fixing it

Comment (iamayushdas) [reply .github/ISSUE_TEMPLATE/bug-report.md]: done

File: .github/PULL_REQUEST_TEMPLATE (hunk 2 above)
Comment (iamayushdas) [reply .github/PULL_REQUEST_TEMPLATE]: done

**Comment by spacewander** [comment]
Need to exclude license check for the template.

**Comment by iamayushdas** [comment]
> Need to exclude license check for the template.

Excluded Sir

**Review by spacewander — APPROVED** [review]

**Pull request merged**